contrib/meson/meson/examples/meson.build

```diff
@@ -1,49 +1,48 @@
 # #############################################################################
-# Copyright (c) 2018-present    lzutao <taolzu(at)gmail.com>
+# Copyright (c) 2018-present        lzutao <taolzu(at)gmail.com>
+# Copyright (c) 2022-present        Tristan Partin <tristan(at)partin.io>
 # All rights reserved.
 #
 # This source code is licensed under both the BSD-style license (found in the
 # LICENSE file in the root directory of this source tree) and the GPLv2 (found
 # in the COPYING file in the root directory of this source tree).
 # #############################################################################
 
-lz4_root_dir = '../../../..'
+examples = {
+  'printVersion': {
+    'sources': files(lz4_source_root / 'examples/printVersion.c'),
```

Comment:
This seems to be optimized for a case that won't come up. I think you can make this a lot simpler.

```meson

examples = {
  'printVersion': 'printVersion.c',
}
```
And later on...